crates/uv/tests/common/mod.rs

```diff
@@ -488,6 +488,10 @@ impl TestContext {
         let mut command = std::process::Command::new(get_bin());
         command.arg("tool").arg("install");
         self.add_shared_args(&mut command);
+        command.env(
```

Comment:
This feels a bit weird, but `add_shared_args` adds the `exclude-newer` arg anyways so it had to be unset. I wasn't able to install `fastapi==0.111` otherwise.

Reply:
Hm that's weird. We should fix that in general.

Fyi there's a method to clear an environment variable, which would be better practice here.

Regardless, we can't merge with this because the transitive dependency versions will change in the future and the snapshot will break — can you install a different version of FastAPI that's available relative to our default exclude newer?

Reply:
> Fyi there's a method to clear an environment variable, which would be better practice here.

The `exclude-newer` is set as an arg in the `add_shared_args` function. I didn't see any method to clear an arg, which is why I ended up overriding with an env variable instead.

>  can you install a different version of FastAPI that's available relative to our default exclude newer?

Unfortunately the previous version`fastapi==0.110` doesn't include `fastapi-cli` as a dependency so it doesn't list any executables. Do you know of any other library that could be used here instead

Reply:
I've reverted this change and switched to a static date in the test itself. What do you think of this approach?

Reply:
Ah okay we need the newer version. Hm. I don't have any good alternatives. A newer static date is okay for now.

> The exclude-newer is set as an arg in the add_shared_args function. I didn't see any method to clear an arg

Ah. Oops, I see.